cron/calculateAssessmentQuestionStats.sql

```diff
@@ -1,4 +1,3 @@
-SELECT
-    assessment_questions_calculate_stats_for_assessment(a.id)
-FROM
-    assessments AS a;
+-- BLOCK select_assessments
+SELECT *
```

Comment:
Do we want a smarter query that only updates on the assessments that have had data changed since the last time stats were run? Or is that not worth it if it churns over all the assessments in negligible time?